### ISSUE TITLE
QSelect options not announced by screen reader

### ISSUE THREAD

**Issue by m-s-c-h** [quasarframework/quasar#14811]
### What happened?

When navigating through the options of a QSelect, the screen reader doesn’t announce anything.
This worked fine in Quasar 2.8.3, but stopped working with Quasar 2.8.4 and is still broken with (current) version 2.10.1.
Tested on Windows (NVDA) and Linux (Orca).

### What did you expect to happen?

When navigating through the options list, the options are announced.

### Reproduction URL

https://codepen.io/m-s-c-h/pen/OJEXxMG

### How to reproduce?

1. Open the provided URL
2. Enable screen reader
3. Tab into the QSelect using the keyboard
5. Press enter or down to open the options list
6. Use the up/down to navigate through the options -> screen reader does NOT announce the options

### Flavour

Quasar CLI with Webpack (@quasar/cli | @quasar/app-webpack)

### Areas

Components (quasar)

### Platforms/Browsers

Firefox, Chrome, Microsoft Edge

**Comment by pdanpdan** [comment]
Please follow https://github.com/quasarframework/quasar/issues/13576
